e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -377,4 +377,27 @@ test.describe('Notebook entry tests', () => {
         expect.soft(await sanitizedLink.count()).toBe(1);
         expect(await unsanitizedLink.count()).toBe(0);
     });
+    test('can export notebook as text', async ({ page }) => {
+        await nbUtils.enterTextEntry(page, `Foo bar entry`);
+        // Click on 3 Dot Menu
+        await page.locator('button[title="More options"]').click();
+        const downloadPromise = page.waitForEvent('download');
+
+        await page.getByRole('menuitem', { name: /Export Notebook as Text/ }).click();
+
+        await page.getByRole('button', { name: 'Save' }).click();
+        const download = await downloadPromise;
+        const readStream = await download.createReadStream();
+        const exportedText = await streamToString(readStream);
+        expect(exportedText).toContain('Foo bar entry');
```

Comment:
We should expand this test to check against the exact text output of a notebook. We should also have tests for checking notebooks with/without tags, metadata, and snapshots (when applicable).

I don't think the PR should be blocked by this, so let's put in some quick stubs?